controller/users_blackbox_test.go

```diff
@@ -90,6 +98,15 @@ func (s *TestUsersSuite) TestUpdateUserOK() {
 	assert.Equal(s.T(), newImageURL, *result.Data.Attributes.ImageURL)
 	assert.Equal(s.T(), newBio, *result.Data.Attributes.Bio)
 	assert.Equal(s.T(), newProfileURL, *result.Data.Attributes.URL)
+	updatedContextInformation := result.Data.Attributes.ContextInformation
```

Comment:
Is there any test when ContextInformation == nil ? To see that everything is still working as expected